ccd-definition/CaseField/CaseFieldLRspec-CUI-nonprod.json

```diff
@@ -0,0 +1,101 @@
+[
```

Comment:
**respondent1FullAdmitPaidLabelSpec** and **applicant1FullAdmitConfirmAmountPaidSpec** are CIV-6176 changes